librarymanagement/src/main/scala/sbt/librarymanagement/RichUpdateReport.scala

```diff
@@ -29,17 +32,27 @@ final class RichUpdateReport(report: UpdateReport) {
   def select(artifact: ArtifactFilter): Seq[File] = select(configurationFilter(), moduleFilter(), artifact)
 
   private[this] def select0(f: DependencyFilter): Seq[File] =
-    for (cReport <- report.configurations; mReport <- cReport.modules; (artifact, file) <- mReport.artifacts if f(cReport.configuration, mReport.module, artifact)) yield {
-      if (file == null) sys.error("Null file: conf=" + cReport.configuration + ", module=" + mReport.module + ", art: " + artifact)
+    for {
+      cReport <- report.configurations
+      mReport <- cReport.modules
+      (artifact, file) <- mReport.artifacts
+      if f(cReport.configuration, mReport.module, artifact)
+    } yield {
+      if (file == null)
```

Comment:
I think we should either use `Option(file).orElse { ... }` or put `{ .. }` around `sys.error`.